src/app/shared/form-controls/material-autocomplete/material-autocomplete.component.html

```diff
@@ -0,0 +1,42 @@
+<div data-jasmine="formControl" data-cy="formFieldWrapper" [formGroup]="form">
+  <mat-form-field
+    attr.data-cy="formfield_{{ controlProperties.key }}"
+    *ngIf="controlProperties"
+    style="width: 100%"
+    appearance="fill"
+  >
+    <mat-label>{{ controlProperties.label }}</mat-label>
+    <input
+      matInput
+      [matAutocomplete]="auto"
+      [formControlName]="controlProperties.key"
+      [placeholder]="controlProperties.placeholder"
+    />
+
+    <button
+      *ngIf="this.form.controls[this.controlProperties.key].value"
+      matSuffix
+      mat-icon-button
+      aria-label="Clear"
+      data-jasmine="clearInputButton"
+      (click)="clearSelection()"
+    >
+      <mat-icon>close</mat-icon>
+    </button>
+
+    <mat-autocomplete
+      [panelWidth]="400"
+      #auto="matAutocomplete"
+      (optionSelected)="onSelected()"
+      [displayWith]="displayWith"
```

Comment:
The property and the method `displayWith` are not required.

Reply:
Ah yes. Thanks.